src/database/models/extra_functionality_movie.py

```
@@ -0,0 +1,47 @@
+from sqlalchemy import Integer, Boolean, ForeignKey, UniqueConstraint
+from sqlalchemy.orm import relationship, Mapped, mapped_column
+
+from .movies import MovieModel
+from .accounts import UserModel
+from database.models.base import Base
+
+
+class MovieLike(Base):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)